tensorboard/plugins/profile/pod_viewer/pod_viewer_dashboard/pod-viewer-dashboard.ts

```diff
@@ -56,58 +79,83 @@ Polymer({
       type: Array,
       value: null,
     },
-    _stepBreakdownEle: {
+    _stepBreakdownItems: {
       type: Array,
-    },
-    _channelEle: {
+      value: () => [
+        {key: 'highFlopsComputeUs', label: 'High flops compute'},
+        {key: 'lowFlopsComputeUs', label: 'Low flops compute'},
+        {key: 'hostInfeedDurationUs', label: 'Infeed'},
+        {key: 'hostOutfeedDurationUs', label: 'Outfeed'},
+        {key: 'crsDurationUs', label: 'All reduce'},
+        {key: 'sendDurationUs', label: 'Send'},
+        {key: 'recvDurationUs', label: 'Recv'},
+      ],
+    },
+    _channelItem: {
       type: Array,
+      value: () => [{key: 'durationUs', label: 'Duration (us)'}],
     },
-    _allReduceEle: {
+    _allReduceItem: {
       type: Array,
+      value: () => [{key: 'durationUs', label: 'Duration (us)'}],
     },
     _stepBreakdownFunc: {
       type: Object,
```

Comment:
I believe you can have a type of `Function`.

Reply:
It seems Function is not available in Polymer 2. Getting bazel compile errors: argument of type ... is not assignable to type 'new () => Base'

Reply:
This still reporting the same error. 
../../../../../../../../../../../../../../tmp/tmpbrHT1n/pod-viewer/pod-viewer-dashboard.ts(15,9): error TS2345: Argument of type '{ is: string; properties: { data: { type: ObjectConstructor; observer: string; }; activeDetails: ...' is not assignable to parameter of type 'Base | (new () => Base)'.
  Type '{ is: string; properties: { data: { type: ObjectConstructor; observer: string; }; activeDetails: ...' is not assignable to type 'new () => Base'.
    Type '{ is: string; properties: { data: { type: ObjectConstructor; observer: string; }; activeDetails: ...' provides no match for the signature 'new (): Base'.

Reply:
The examples in this repo put the js files in the .html file, perhaps bypasses the check.